install/installer/cmd/testdata/render/minimal/output.golden

```diff
@@ -3333,7 +3333,12 @@ data:
         "counterMetrics": [
           {
             "name": "gitpod_supervisor_frontend_error_total",
-            "help": "Total count of web browser error in workspace",
+            "help": "Total count of suprvisor frontend error in workspace",
+            "labels": null
+          },
+          {
+            "name": "gitpod_supervisor_frontend_window_total",
+            "help": "Total count of suprvisor frontend client",
```

Comment:
`Total count of supervisor frontend clients.`

Actually client sounds better, let's rename to `gitpod_supervisor_frontend_client_total`